addons/website/tests/test_attachment.py

```diff
@@ -51,3 +55,16 @@ def test_01_type_url_301_image(self):
 
         req = self.opener.get(base + '/web/image/test.an_image_redirect_301', allow_redirects=True)
         self.assertEqual(req.status_code, 200)
+
+    def test_02_image_quality(self):
+        IrAttachment = self.env['ir.attachment']
+        base = "http://%s:%s" % (HOST, config['http_port'])
+        req = self.opener.get(base + '/web/image/website.s_banner_default_image')
+        IrAttachment.create({
+            'public': True,
+            'name': 's_banner_default_image.jpg',
+            'type': 'binary',
+            'res_model': 'ir_ui_view',
+            'datas': base64.b64encode(req.content),
+        })
```

Comment:
Why do you need to create an attachment here ?
I did a test with a breakpoint before your create, when I launch the test and do `IrAttachment.search([('name', '=', 's_banner_default_image.jpg')])` I got an attachment already.. maybe there is something I missed ?

Reply:
The "default" attachment does not have `res_model` set to `ir_ui_view`. I think this prevents it from being available for selection in the media dialog. We therefore create this new one. Maybe I should use a dedicated name instead, to make it more clear ? (e.g. `test_image_quality.jpg`)

Reply:
Ok but then why does your test tour needs to replace the existing image in s_text_image ? Is there a different behavior with default images than in upladed images ? I had a quick look and the quality option is there on default image.
![image](https://user-images.githubusercontent.com/78849981/229843067-555f91d0-18e3-4301-9ee0-87b43e194f58.png)

Reply:
I adapted the test to rely on the default image instead.